build/lib/playground/createSvelte5Playground.js

```diff
@@ -0,0 +1,55 @@
+import path from "node:path";
+
+export default function createSvelte5Playground() {
+  const BASE_URL = "https://svelte.dev/playground?version=5#";
```

Comment:
To get rid of the "Hello World".
```suggestion
  const BASE_URL = "https://svelte.dev/playground/untitled?version=5#";
```

Open issue about encoding title:
- https://github.com/sveltejs/svelte.dev/issues/478